src/async_client.rs

```diff
@@ -218,10 +218,14 @@ impl Client {
         let dls = self.crate_downloads(name);
         let owners = self.crate_owners(name);
         let reverse_dependencies = self.crate_reverse_dependencies(name);
+        // TODO:
+        // let (reverse_dependencies, reverse_dependencies_versions, reverse_dependencies_total) = self.crate_reverse_dependencies(name);
```

Comment:
Do you need help in implementing this?

Reply:
This should be fixed already. I'm sorry, I think I made a mess of the review process...